frontend/javascripts/oxalis/controller/scene_controller.js

```diff
@@ -339,6 +340,12 @@ class SceneController {
     // though they are all looking at the same scene, some
     // things have to be changed for each cam.
 
+    const { tdViewDisplayPlanes, tdViewDisplayDatasetBorders } = Store.getState().userConfiguration;
+
+    // Only set the visibility of the dataset bounding box for the TDView.
+    // This has to happen before updateForCam is called as otherwise cross section visibility
```

Comment:
Can you remind me what the cross sections are? I found the definition in `Cube` but couldn't really find out what these are.

Reply:
The cross sections exist so that the dataset bounding box is always visible in each of the three 2D views (XY, XZ, YZ), although the bounding box is actually just a cuboid wireframe and not solid.
They are the dashed line in this screenshot:

![](https://qph.fs.quoracdn.net/main-qimg-78294477fd4255bae6db27f8f235b4fe.webp)

Reply:
Ah, thanks for the refresher :+1: I think I understood what's happening now :)